cql-wih/src/test/java/io/elimu/a2d2/cql/CqfClientWIHTest.java

```diff
@@ -0,0 +1,43 @@
+package io.elimu.a2d2.cql;
+
+import java.util.List;
+import java.util.Map;
+
+import org.drools.core.process.instance.impl.WorkItemImpl;
+import org.junit.Assert;
+import org.junit.Test;
+
+import io.elimu.a2d2.cdsresponse.entity.Card;
+
+public class CqfClientWIHTest {
+
+	@Test
+	public void testClientCall() throws Exception {
+		PlanDefCdsWorkItemHandler handler = new PlanDefCdsWorkItemHandler();
+		WorkItemImpl workItem = new WorkItemImpl();
+		workItem.setParameter("fhirServerUrl", "https://cqf-internal.elimuinformatics.com/fhir");
```

Comment:
@marianbuenosayres We'll be tearing down the cqf-internal server soon. Should we rely on that server for tests?

Reply:
@boxwala we don't have another public one to test this from yet